pydantic/errors.py

```diff
@@ -303,6 +303,8 @@ class ListMinLengthError(PydanticValueError):
     msg_template = 'ensure this value has at least {limit_value} items'
 
     def __init__(self, *, limit_value: int) -> None:
+        if limit_value == 1:
+            self.msg_template = 'ensure this value has at least {limit_value} item'
```

Comment:
I didn't reverse it (e.g. `msg = 'count {limit_value} item' + 's' if limit_value > 1 else ''`) because I thought this was more readable. Would be happy to create a `PluralItemsMixin` or something if preferred 😊 . I'm not doing a ton of Python development these days so any feedback / improvement ideas are welcome.